packages/reference/src/components/ResourceEntityErrorCard/FunctionInvocationErrorCard.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { Flex, Text, TextLink } from '@contentful/f36-components';
+
+import { MissingEntityCard } from '..';
+
+import { ErrorCircleOutlineIcon, ExternalLinkIcon } from '@contentful/f36-icons';
+
+import { useResourceProvider } from '../../common/EntityStore';
+
+type FunctionInvocationErrorLogLinkProps = {
+  organizationId: string;
+  appDefinitionId: string;
+  functionId?: string;
+  status: 'error' | 'success' | 'loading';
+};
+
+function FunctionInvocationErrorLogLink(props: FunctionInvocationErrorLogLinkProps) {
+  const functionLink = `/account/organizations/${props.organizationId}/apps/definitions/${props.appDefinitionId}/functions/${props.functionId}/logs`;
+
+  const children = (
```

Comment:
As it's only used once, you can add it directly below ;)
Then it's easier to follow the code 😅